md_eval_function.m

```diff
@@ -16,7 +16,12 @@
             md_funcs{s}{d},params,t);
     end
     
+    % Captain! This time multiplier can just be factored out. You factor out time multiplier
+    % from inside combine_dimensions and apply the next two lines as axpys later.
     time_multiplier = md_funcs{s}{num_dimensions+1}(t);
+    %Captain! All these functions use the same time multiplier
+    %Captain! All combine dimensions outputs are stored in a vector and scaled by the time
```

Comment:
yes, there is a time multiplier not related to a particular dimension, but each dimension also has its own time multiplier

Reply:
these are more semi-coherent annotations I put in the code for myself as I was figuring out how it worked